app/services/contents/builder.rb

```diff
@@ -7,13 +7,14 @@ def self.call(...)
       new(...).call
```

Comment:
I'm fuzzy on how we're distinguishing mappers from builders. What do you see as the difference between them?

Reply:
I'm my mind, a builder creates AR objects, whereas a mapper just maps without persisting anything.

But I don't feel strongly about this if you have other thoughts.

Reply:
That's a good distinction. I don't care too much whether we call our builder classes "builders" or "data mappers" or whatever, as long as we developers can keep the distinction in our heads. I might do a short spike on teasing these apart, and we can either keep it or throw it away.